radixdlt/src/main/java/com/radixdlt/consensus/BFTEventReducer.java

```diff
@@ -153,43 +154,43 @@ public void processLocalSync(Hash vertexId) {
 
 	@Override
 	public void processVote(Vote vote) {
-		log.trace("{}: VOTE: Processing {}", this.getShortName(), vote);
+		log.trace("{}: VOTE: Processing {}", this::getShortName, () -> vote);
 		// accumulate votes into QCs in store
 		this.pendingVotes.insertVote(vote, this.validatorSet).ifPresent(qc -> {
-			log.info("{}: VOTE: Formed QC: {}", this.getShortName(), qc);
+			log.trace("{}: VOTE: Formed QC: {}", this::getShortName, () -> qc);
 			if (vertexStore.syncToQC(qc, vertexStore.getHighestCommittedQC(), vote.getAuthor())) {
```

Comment:
I think it would be really convenient to see when the node becomes unsynched, and then resynched here.  Something like:
```
	if (vertexStore.syncToQC(qc, vertexStore.getHighestCommittedQC(), vote.getAuthor())) {
		if (!synchedLog) {
			log.info("{}: VOTE: QC Synced: {}", this::getShortName, () -> qc);
			synchedLog = true;
		}
		processQC(qc);
	} else {
		if (synchedLog) {
			log.info("{}: VOTE: QC Not synced: {}", this::getShortName(), () -> qc);
			synchedLog = false;
		}
		unsyncedQCs.put(qc.getProposed().getId(), qc);
	}
```
...so we can track how frequently nodes become synced and unsynced in logs at a reasonably high log level, without flooding logs.

Reply:
Shall we add the same logging to processLocalSync?

Reply:
Could do, yes.